On macos 10.15:

```
builder for '/nix/store/xafbna5861brq4l8qp6hixjd1ms9py1z-gigalixir-1.2.3.drv' failed with exit code 1; last 10 log lines:
      resp = self.send(prep, **send_kwargs)
    File "/nix/store/cvv6agb0fs9cp9bpqcivm3zkp50a4baq-python3.8-requests-2.26.0/lib/python3.8/site-packages/requests/sessions.py", line 655, in send
      r = adapter.send(request, **kwargs)
    File "/nix/store/cvv6agb0fs9cp9bpqcivm3zkp50a4baq-python3.8-requests-2.26.0/lib/python3.8/site-packages/requests/adapters.py", line 516, in send
      raise ConnectionError(e, request=request)
  requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.rollbar.com', port=443): Max retries exceeded with url: /api/1/item/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x10515caf0>: Failed to establish a new connection: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)'))
  ERROR    gigalixir-cli:__init__.py:88 ('Connection aborted.', OSError(57, 'Socket is not connected'))
  =========================== short test summary info ============================
  FAILED test/test_gigalixir.py::test_rollback_without_version - assert '' == '...
  ========================= 1 failed, 53 passed in 1.56s =========================
```

Not sure why this doesn't fail on linux (doesn't run?).

--- a/pkgs/development/python-modules/gigalixir/default.nix
+++ b/pkgs/development/python-modules/gigalixir/default.nix
@@ -1,0 +1,55 @@
+{ buildPythonApplication
+, click
+, fetchPypi
+, git
+, httpretty
+, lib
+, qrcode
+, pygments
+, pyopenssl
+, pytestCheckHook
+, requests
+, rollbar
+, stripe
+, sure
+}:
+
+buildPythonApplication rec {
+  pname = "gigalixir";
+  version = "1.2.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1b7a9aed7e61a3828f5a11774803edc39358e2ac463b3b5e52af267f3420dc66";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py --replace "'pytest-runner'," ""
+  '';
+
+  propagatedBuildInputs = [
+    click
+    requests
+    stripe
+    rollbar
+    pygments
+    qrcode
+    pyopenssl
+  ];
+
+  checkInputs = [
+    httpretty
+    sure
+    pytestCheckHook
+    git
+  ];
+
+  pythonImportsCheck = [ "gigalixir" ];
+
+  meta = with lib; {
+    description = "Gigalixir Command-Line Interface";
+    homepage = "https://github.com/gigalixir/gigalixir-cli";
+    license = licenses.mit;
+    maintainers = with maintainers; [ superherointj ];
+  };
+}

--- a/pkgs/development/python-modules/rollbar/default.nix
+++ b/pkgs/development/python-modules/rollbar/default.nix
@@ -1,0 +1,47 @@
+{ aiocontextvars
+, blinker
+, buildPythonPackage
+, fetchPypi
+, httpx
+, lib
+, mock
+, pytestCheckHook
+, requests
+, six
+, unittest2
+, webob
+}:
+
+buildPythonPackage rec {
+  pname = "rollbar";
+  version = "0.16.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "aa3b570062dd8dfb0e11537ba858f9e1633a604680e062a525434b8245540f87";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    six
+  ];
+
+  checkInputs = [
+    webob
+    blinker
+    unittest2
+    mock
+    httpx
+    aiocontextvars
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ "rollbar" ];
+
+  meta = with lib; {
+    description = "Error tracking and logging from Python to Rollbar";
+    homepage = "https://github.com/rollbar/pyrollbar";
+    license = licenses.mit;
+    maintainers = with maintainers; [ superherointj ];
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -24080,6 +24080,8 @@ with pkgs;
 
   fnott = callPackage ../applications/misc/fnott { };
 
+  gigalixir = with python3Packages; toPythonApplication gigalixir;
+
   go-libp2p-daemon = callPackage ../servers/go-libp2p-daemon { };
 
   go-motion = callPackage ../development/tools/go-motion { };

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -7945,6 +7945,8 @@ in {
 
   rokuecp = callPackage ../development/python-modules/rokuecp { };
 
+  rollbar = callPackage ../development/python-modules/rollbar { };
+
   roman = callPackage ../development/python-modules/roman { };
 
   roombapy = callPackage ../development/python-modules/roombapy { };

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -2960,6 +2960,8 @@ in {
 
   gidgethub = callPackage ../development/python-modules/gidgethub { };
 
+  gigalixir = callPackage ../development/python-modules/gigalixir { };
+
   gin-config = callPackage ../development/python-modules/gin-config { };
 
   gios = callPackage ../development/python-modules/gios { };

